### PR TITLE
dev -> main for 3.1.1

### PR DESCRIPTION
**Pull request by mirpedrol**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +42,7 @@
 - Add `manifest.contributors` to `nextflow.config` ([#3311](https://github.com/nf-core/tools/pull/3311))
 - Update template components ([#3328](https://github.com/nf-core/tools/pull/3328))
 - Template: Remove mention of GRCh37 if igenomes is skipped ([#3330](https://github.com/nf-core/tools/pull/3330))
+- Be more verbose in approval check action ([#3338](https://github.com/nf-core/tools/pull/3338))
```
Comment (mirpedrol) [inline CHANGELOG.md:45]: entry added to wrong version

File: CHANGELOG.md [hunk 2]
```diff
@@ -65,6 +85,9 @@
 - Don't break gitpod.yml with template string ([#3332](https://github.com/nf-core/tools/pull/3332))
 - rocrate: remove duplicated entries for name and version ([#3333](https://github.com/nf-core/tools/pull/3333))
 - rocrate: Update crate with version bump and handle new contributor field ([#3334](https://github.com/nf-core/tools/pull/3334))
+- set default_branch to master for now ([#3335](https://github.com/nf-core/tools/issues/3335))
+- Set git defaultBranch to master in sync action ([#3337](https://github.com/nf-core/tools/pull/3337))
+- Add verbose mode to sync action ([#3339](https://github.com/nf-core/tools/pull/3339))
```
Comment (mirpedrol) [inline CHANGELOG.md:90]: wrong version

File: nf_core/pipeline-template/nextflow.config [hunk 3]
```diff
@@ -77,11 +77,6 @@ params {
 includeConfig 'conf/base.config'
 {%- else %}
 
-{% if modules -%}
-// Load modules.config for DSL2 module specific options
-includeConfig 'conf/modules.config'
```
Comment (mashehu) [inline nf_core/pipeline-template/nextflow.config:82]: Are we now not loading modules 
.config anywhere anymore?

**Review by mashehu — APPROVED** [review]

File: nf_core/pipeline-template/nextflow.config (hunk 3 above)
Comment (mirpedrol) [reply nf_core/pipeline-template/nextflow.config:82]: it is at the end https://github.com/nf-core/tools/pull/3369/files#diff-957ebc7549e67b6aa1df19ae402551fee3e2e06afbc7c563424a4515b1434627R337

**Pull request merged**